core/test/qa_Scheduler.cpp

```diff
@@ -259,8 +262,8 @@ ENABLE_REFLECTION_FOR_TEMPLATE(LifecycleSource, out);
 
 template<typename T>
 struct LifecycleBlock : public gr::Block<LifecycleBlock<T>> {
-    gr::PortIn<T>  in{};
-    gr::PortOut<T> out{};
+    gr::PortIn<T>                in{};
+    gr::PortOut<T, gr::Optional> out{};
```

Comment:
Great job identifying the problem! We should remember to mark ports as `Optional` if they are not in use.